scanner/src/main/kotlin/ScanContext.kt

```diff
@@ -21,6 +21,7 @@ package org.ossreviewtoolkit.scanner
 
 import org.ossreviewtoolkit.model.LicenseFinding
 import org.ossreviewtoolkit.model.OrtResult
+import org.ossreviewtoolkit.model.Package
```

Comment:
commit-msg: Please provide a rationale _why_ some package scanners need to know that.